packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -699,6 +699,24 @@ export default class Meetings extends WebexPlugin {
     }
   }
 
+  /**
+   * API to toggle TCP reachability, needs to be called before webex.meetings.register()
+   * @param {Boolean} newValue
+   * @private
+   * @memberof Meetings
+   * @returns {undefined}
+   */
+  private _toggleTcpReachability(newValue: boolean) {
```

Comment:
yes, this function is a bit weird, but I just copied the approach from similar other existing ones